src/task01.java

```diff
@@ -0,0 +1,93 @@
+
+interface control {
+
+    void speed();
+}
+
+abstract class Vehicle implements control{
+
+    String type;
+    public Vehicle(String typ)
+    {
+        this.type = typ;
+    }
+    abstract void info();
+}
+
+class Road extends Vehicle{
+
+    public Road()
+    {
+        super("Road");
+    }
+
+    static String owner_name, model_no;
```

Comment:
why static ? We use static when we are in need to share the single copy of method/data member among all objects of the class. Here we can see that , all Road's object (two_wheller) may have different owners and model no of vehicle.